### PR TITLE
Adding native AccelerationMinimizer support

### PR DESCRIPTION
**Pull request by keenon**
As I've been using the AddBiomechanics data, especially the computed torque data, I've noticed that the histograms of joint torques are pretty untrustworthy. After a bit of investigation, it turns out that our low pass filter isn't getting rid of all the high frequency noise in the position signal, which means that the accelerations (and consequently torques) still have a ton of noise in the signal, which is large enough that it can ruin my histograms of joint torques to try to size exo motors.

After a bit of experimentation, I've landed on a least-squares smoother that minimizes accelerations while regularizing that the signal still stays near the original signal. My Python implementation is slow, and doesn't scale to lots of time series, so I wrote a nice optimized C++ version.

### CODE REVIEW

File: python/nimblephysics.egg-info/PKG-INFO [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Metadata-Version: 2.1
 Name: nimblephysics
-Version: 0.10.42.1
+Version: 0.10.43
```
Comment (nickbianco) [inline python/nimblephysics.egg-info/PKG-INFO:3]: Should this be `0.10.45`?

File: dart/utils/AccelerationMinimizer.cpp [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+// #include <unsupported/Eigen/IterativeSolvers>
+
+#include "dart/math/MathTypes.hpp"
+
+namespace dart {
+namespace utils {
+
+/**
+ * Create (and pre-factor) a smoother that can remove the "jerk" from a time
+ * seriese of data.
```
Comment (nickbianco) [inline dart/utils/AccelerationMinimizer.cpp:17]: "series"

**Review by nickbianco — APPROVED** [review]
I noticed a couple of minor things, but otherwise LGTM.

File: dart/utils/AccelerationMinimizer.cpp (hunk 2 above)
Comment (keenon) [reply dart/utils/AccelerationMinimizer.cpp]: Ok, pushed fixes!

**Pull request merged**